internal/app/handlers/routes.go

```diff
@@ -0,0 +1,23 @@
+package handlers
+
+import (
+	"github.com/go-chi/chi/v5"
+	"net/http"
+)
+
+// AppRoutes маршруты приложения
+func AppRoutes() chi.Router {
+	r := chi.NewRouter()
+	// Код ошибки по умолчанию, согласно ТЗ
+	r.MethodNotAllowed(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("method not expect\n"))
+	})
+
+	r.Route("/", func(r chi.Router) {
+		r.Post("/", DecodeHandler)
+		r.Get("/{id}", EncodeHandler)
+	})
```

Comment:
Мне кажется тут можно опустить  `r.Route("/", func(r chi.Router)`, так как определение маршрута полезно в случае, если у маршрута есть общий префикс, который можно будет опустить. Например  `r.Route("/api", func(r chi.Router)`

```suggestion
            r.Post("/", DecodeHandler)
            r.Get("/{id}", EncodeHandler)
```